js/src/table.ts

```diff
@@ -238,17 +238,30 @@ export class Table<T extends TypeMap = any> {
      *
      * @returns An Array of Table rows.
      */
-    public toArray() {
+    public toArray(): Array<Struct<T>['TValue']> {
```

Comment:
Does this improve the type returned by this method or just make the type explicit?

Reply:
It just makes it explicit. I added it to know precisely what `toArrayView()` was supposed to return. And also because I like explicit types 😉

Reply:
Unlike in other languages, explicitly annotating return types is somewhat of an anti-pattern in TypeScript. Unless there's a special case where we need to inform the compiler what the real return type is, I'm with @domoritz that we should default to letting the compiler determine the type.

Reply:
Ok, no problem, I removed it in 4b706a9.